cms/server/admin/templates/fragments/submission_row.html

```diff
@@ -46,6 +46,7 @@
         <table class="nested bordered">
           <thead>
             <tr>
+              <th>#</th>
```

Comment:
For some reason this file (`cms/server/admin/templates/fragments/submission_row.html`) doesn't need to be patched. I think that the table is taken from the ScoreType HTML template. I am not sure if this file is even used anymore.

May we just remove this file?

/cc @stefano-maggiolo